app/templates/components/events/view/overview/general-info.hbs

```diff
@@ -15,7 +15,12 @@
     <tbody>
       <tr>
         <td><strong>{{t 'No. of Revisions'}}</strong></td>
-        <td>6 {{t '(view revisions)'}}</td>
+        <td>
+          6 
+          <button onclick={{action 'openModal'}} class="mini circular ui button">
```

Comment:
let this be a link, like before.

Reply:
Keeping it a link with onclick give me a lint error saying something about interactive element applied to non interactive element.

Reply:
use `role="button"` as an attribute.